ci/checks/ensure-version-bumped.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail -u
+
+exit_code=0
+
+last_tag=$(git describe --tags --abbrev=0)
+last_tag_version=${last_tag:1}
+current_version=$(grep '__version__ = ' git_machete/__init__.py | cut -d\' -f2)
+
+if [[ "$(printf "$current_version\n$last_tag_version" | sort -V -r | head -n 1)" == "$last_tag_version" ]]; then
```

Comment:
Nit, for slightly more clarity (IMO):
```suggestion
if [[ "$(printf "$current_version\n$last_tag_version" | sort --version-sort | tail -n 1)" == "$last_tag_version" ]]; then
```

(expand the lesser-known `-V` flag to its long name + remove the reversal)